extensions/wikia/Captcha/CaptchaHooks.class.php

```diff
@@ -58,4 +58,12 @@ public static function APIGetAllowedParams( &$module, &$params ) {
 	public static function APIGetParamDescription( &$module, &$desc ) {
 		return Factory\Module::getInstance()->APIGetParamDescription( $module, $desc );
 	}
+
+	// Temporarily load captcha library on every page until we have on demand loading set up.
+	// Remove with https://wikia-inc.atlassian.net/browse/SOC-288
```

Comment:
I think the wording in SOC-288 should be updated to mention the new plan, which is to load the `captcha_js` AM group server side via those hooks as well as on the UserSignup page.